bubblewrap.c

```diff
@@ -219,6 +220,7 @@ usage (int ecode, FILE *out)
            "    --info-fd FD                 Write information about the running container to FD\n"
            "    --new-session                Create a new terminal session\n"
            "    --die-with-parent            Kills with SIGKILL child process (COMMAND) when bwrap or bwrap's parent dies.\n"
+           "    --no-reaper                  Do not install a reaper process with PID=1\n"
```

Comment:
(Sorry to bikeshed, but OTOH the command line *is* the bwrap "API" and it makes sense to consider it carefully I think)